### DISCUSSION TITLE
Exam-network, long loading time

### DISCUSSION THREAD

**Discussion by MarkoMard** [SafeExamBrowser/seb-win-refactoring#253] [Q&A] [answered]
We have a very limited exem vlan and guys are reporting that it takes a long time for seb to get to our login screen. In normal edu net all is speedy. It seems that there are connections trying to access ip's 52.152.110.14 and 93.184.220.29. Are they needed? Should we grant access to them?

**Comment by danschlet** [comment, marked as answer]
If you [look up those IP addresses](https://whatismyipaddress.com), you can see that they belong to Microsoft Azure and Verizon Business. I can assure you that SEB doesn't contact such addresses (see also our [privacy statement](https://safeexambrowser.org/about_overview_en.html#privacy-statement)). "SEB is not connecting to any cloud service or server, besides the web server which you enter as "Start URL" in your settings."

You need to find out what in your web application contacts such servers.

**Reply by MarkoMard** [reply]
Thanks, we'll continue :)
